tasks/task_manager.hh

```diff
@@ -48,7 +48,7 @@ private:
     modules _modules;
     config _cfg;
     seastar::abort_source _as;
-    optimized_optional<abort_source::subscription> _abort_subscription;
+    optimized_optional<seastar::abort_source::subscription> _abort_subscription;
```

Comment:
nitpick: do we want to add "seastar::" in all these cases of abort_source - isn't it easier to remove "seastar::" from all of them?

Reply:
There is a member function called abort_source, so we can't refer to the global type abort_source::subscription without full qualification. This isn't about having consistent names.